test/integration/install/install_test.go

```diff
@@ -144,7 +144,7 @@ func TestInstallOrUpgradeCli(t *testing.T) {
 		cmd  = "install"
 		args = []string{
 			"--controller-log-level", "debug",
-			"--proxy-version", TestHelper.GetVersion(),
+			"--set proxy.image.version", TestHelper.GetVersion(),
```

Comment:
You'll have to follow the same format everywhere I think, where you append an `=`